docs/content/tutorials/mongodb-tutorial.md

```diff
@@ -0,0 +1,277 @@
+# Streaming ETL from MongoDB to Elasticsearch
+
+1. Create `docker-compose.yml` file using following contents: 
+
+```
+version: '2.1'
+services:
+  postgres:
+    image: debezium/example-postgres:1.1
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=1234
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+  mysql:
+    image: debezium/example-mysql:1.1
+    ports:
+      - "3306:3306"
+    environment:
+      - MYSQL_ROOT_PASSWORD=123456
+      - MYSQL_USER=mysqluser
+      - MYSQL_PASSWORD=mysqlpw
```

Comment:
Could you remove all content about postgres and mysql because they are not needed?

Reply:
ok

Reply:
Fixed.